applications/sdp/gpio/src/main.c

```diff
@@ -13,6 +15,16 @@
 #include <hal/nrf_vpr_csr_vio.h>
 #include <haly/nrfy_gpio.h>
 
+#define HRT_IRQ_PRIORITY      2
+#define HRT_VEVIF_GPIO_CLEAR  17
+#define HRT_VEVIF_GPIO_SET    18
+#define HRT_VEVIF_GPIO_TOGGLE 19
```

Comment:
Maybe these names should include IRQ, for example, `HRT_IRQ_VEVIF_GPIO_CLEAR`? Right now, just by reading these lines, it is difficult to tell what those are for.

Reply:
My aim here was to show that VEVIF index is not identical to IRQ number.
Using `HRT_IRQ_...` could be confusing when there is `VEVIF_IRQN` macro.
Maybe `HRT_VEVIF_IDX_GPIO_CLEAR` would be better?

Reply:
Ok, my bad, I thought those were IRQ numbers. But yes, I think adding `IDX` would make it more clear anyway

Reply:
Done